app/src/main/java/com/github/se/polyfit/data/remote/firebase/PostFirebaseRepository.kt

```diff
@@ -89,6 +90,8 @@ class PostFirebaseRepository(
       completion: (List<Post>) -> Unit,
       geoFire: GeoFire = GeoFire(geoFireRef)
   ) {
+    Log.d("MapRadius", "Radius: $radiusInKm")
+    Log.d("MapCenter", "Center: $centerLatitude, $centerLongitude")
```

Comment:
To avoid congesting the logs maybe a good idea to remove the debug: 
```suggestion

```

Same thing same for the other `log` in this file